io/io/inc/TMemFile.h

```diff
@@ -26,7 +26,7 @@ class TMemFile : public TFile {
       ExternalDataRange_t(const char * start, const size_t size) : fStart(start), fSize(size) {}
    };
 
-private:
+protected:
```

Comment:
This exposes implementation details of `TMemFile`, so I would refrain from this change at all costs.